roles/deploy/defaults/main.yml

```diff
@@ -48,6 +48,9 @@ project_current_path: "{{ project.current_path | default('current') }}"
 # Whether to run `wp core update-db` at end of each deploy
 update_db_on_deploy: true
 
+# Whether to autoload classes from classmap only.
+composer_classmap_authoritative: false
```

Comment:
Are you thinking that `true` isn't safe enough as the default?

Reply:
Correct.
I am running `--classmap-authoritative` to see any of my sites break.

Maybe default to `true` on next major relrease

Reply:
After a second thought, it is very difficult to make a wp plugin that:
- generates classes dynamically, and
- relies on bedrock's composer autoloader

Defaults `composer_classmap_authoritative` to `true`

Ready for review